src/compiler/membership.md

```diff
@@ -154,3 +154,33 @@ If a member or maintainer has been inactive in the compiler for 6 months, then w
 they would like to go to alumni status. If they respond yes or do not respond, they can be placed on
 alumni status.  If they would prefer to remain active, that is also fine, but they will get asked
 again periodically if they continue to be inactive.
+
+### Process: Adding a new team member
+When a potential team member has been nominated by existing members, there is a standard process
+that can be followed by team leads to add the new team member:
+
+1. Contact the nominatees asking if they are interested in joining the team:
```

Comment:
"nominees"?

(apologies, not my mother-tongue)

Reply:
You're absolutely right - rust-lang/rust-forge#864